superset/dashboards/schemas.py

```diff
@@ -121,6 +123,45 @@ class DashboardJSONMetadataSchema(Schema):
     label_colors = fields.Dict()
 
 
+class UserSchema(Schema):
+    id = fields.Int()
+    username = fields.String()
+    first_name = fields.String()
+    last_name = fields.String()
+
+
+class RolesSchema(Schema):
+    id = fields.Int()
+    name = fields.String()
+
+
+class DatasourceSchema(Schema):
+    id = fields.Int()
+    name = fields.String()
+    description = fields.String()
+
+
+class DashboardResponseSchema(Schema):
```

Comment:
maybe `DashboardGetResponseSchema` since responses vary from HTTP method to HTTP method